lib/zlib.js

```diff
@@ -119,8 +121,8 @@ function zlibBufferSync(engine, buffer) {
   buffer = processChunkSync(engine, buffer, engine._finishFlushFlag);
   if (engine._info)
     return { buffer, engine };
-  else
-    return buffer;
+// no need to use else 
+ return buffer;
```

Comment:
Did you run `make lint`? This indentation seems off.

Reply:
yes i did no problem